jupyterhub/spawners.py

```diff
@@ -318,10 +318,10 @@ def get_pod_manifest(self):
         }
         # some annotations are repeated as labels so that the k8s api can filter resources
         self.extra_labels = {
-            RENKU_ANNOTATION_PREFIX + "username": safe_username,
+            RENKU_ANNOTATION_PREFIX + "username": safe_username[:63],
             RENKU_ANNOTATION_PREFIX + "commit-sha": options.get("commit_sha"),
-            RENKU_ANNOTATION_PREFIX + "projectName": options.get("project"),
```

Comment:
Should we keep the project name as an annotation? I understand why you switched to the ID for filtering, but having the name in the annotations could still be useful for us when debugging, etc.

Reply:
This is the list of labels @ableuler. The project name already exists in the annotations. I did not remove that.

Reply:
ah sorry, forgot for a moment that we duplicate all labels into annotations already anyway.